front/src/app/_services/map.service.ts

```diff
@@ -401,6 +431,28 @@ export class MapService {
     return dragAndDropInteraction;
   }
 
+  private createDrawingInteraction(drawingMode?: string) {
+    if (drawingMode === 'Box') {
+      this.drawInteraction = new Draw({
+        source: this.drawingSource,
+        type: GeometryType.CIRCLE,
```

Comment:
Uh? I don't get that, why `CIRCLE` ?

Reply:
Because all rectangles are circles that don't dare to show their curves.

I think it's related to the interaction itself. When you draw a rectangle and a circle, you do the same gestures: click & drag, the only difference is the `geometryFunction`.

I copied it from here: https://openlayers.org/en/latest/examples/draw-shapes.html